fog/view/enclave/impl/src/lib.rs

```diff
@@ -182,6 +193,29 @@ where
         for rec in records {
             store.add_record(&rec.search_key, &rec.payload)?;
         }
+
         Ok(())
     }
+
+    /// Takes in a client's query request and returns a list of query requests
+    /// to be sent off to each Fog View Store shard.
+    fn create_multi_view_store_query_data(
+        &self,
+        client_query: EnclaveMessage<ClientSession>,
+    ) -> Result<Vec<EnclaveMessage<ClientSession>>> {
+        let client_query_bytes = self.ake.client_decrypt(client_query.clone())?;
+
+        let mut encryptors = self.store_encryptors.lock()?;
+        let mut results = Vec::new();
```

Comment:
Let's reserve the expected size
```suggestion
        let mut results = Vec::with_capactiy(encryptors.len());
```

Reply:
Done.